tests/integration/kubernetes/runtimeclass_workloads/k8s-policy-exec-rejected.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2018 Intel Corporation
```

Comment:
```suggestion
# Copyright (c) 2018 Microsoft Corporation
```

Reply:
This is a copy of a file contributed by Intel, with minor changes.

Reply:
I'd **also** add MSFT copyright then, but that's not mandatory / should be needed unless I find something else that would require a re-spin of the PR.

Reply:
Good to know - thanks guys! I made this change while rebasing today.